src/main/java/Frappe.java

```diff
@@ -0,0 +1,182 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Frappe {
+
+    public static Task[] tasks = new Task[100];
+    public static int cur_idx = 0;
+
+    public static void printWelcome() {
+        System.out.println("Hello! I'm Frappe.");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void printUnderscoreLine() {
+        System.out.println(new String(new char[50]).replace("\0", "_")
+        );
+    }
+
+    public static void printTaskAdded(Task task) {
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(task.getPrintOutput());
+        System.out.println("Now you have " + String.valueOf(cur_idx) + (cur_idx == 1 ? " task " : " tasks ") + "in the list.");
+    }
+
+    public static void printInvalidCommand() {
+        System.out.println("Invalid command format. Please try again.");
+    }
+
+    public static void printBye() {
+        System.out.println("Bye. Hope to see you again soon!");
+        printUnderscoreLine();
+    }
+
+    public static void printTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < cur_idx; i++)
+            System.out.println(String.format("%d. ", i + 1) + tasks[i].getPrintOutput());
+    }
+
+    public static void updateTaskComplete(String[] words, Boolean complete) {
```

Comment:
maybe use a prefix for boolean, like isComplete!
```suggestion
    public static void updateTaskComplete(String[] words, Boolean isComplete) {
```